axelrod/tests/unit/test_fingerprint.py

```diff
@@ -325,21 +408,21 @@ def test_pair_fingerprints(self, strategy_pair):
         """
         strategy, probe = strategy_pair
         af = AshlockFingerprint(strategy, probe)
-        data = af.fingerprint(turns=2, repetitions=2, step=0.5,
+        data = af.fingerprint(turns=2, repetitions=2, step=1.0,
```

Comment:
This is an edge case though. Keep it at `0.5` please so that it does test all point creations.